src/main/java/org/opensearch/security/securityconf/impl/v7/InternalUserV7.java

```diff
@@ -44,115 +44,120 @@
 
 public class InternalUserV7 implements Hideable, Hashed, StaticDefinable {
         
-        private String hash;
-        private boolean reserved;
-        private boolean hidden;
-        @JsonProperty(value = "static")
-        private boolean _static;
-        private List<String> backend_roles = Collections.emptyList();
-        private Map<String, String> attributes = Collections.emptyMap();
-        private String description;
-        private List<String> opendistro_security_roles = Collections.emptyList();
-
-        private InternalUserV7(String hash, boolean reserved, boolean hidden, List<String> backend_roles, Map<String, String> attributes) {
-            super();
-            this.hash = hash;
-            this.reserved = reserved;
-            this.hidden = hidden;
-            this.backend_roles = backend_roles;
-            this.attributes = attributes;
-        }
-
```

Comment:
It shouldnt be counted as change. Can you fix this? 
The only changes that should be visible is the one you did. Can you move back to previous version and only change for whatever you made.